api-server/src/homepage/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+</head>
+
+<body>
+  <div class="overlord">
+    <div class="welcome-text">
+      Secret Database Dashboard<span class="blink"> &#9610;</span>
+    </div>
+
+    <div class="buttons-container">
+      <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">
+        <button type="button" class="death-button">Delete Database</button>
+      </a>
+      <a>
+        <button type="button" class="happy-button" onclick="retreat()">Retreat</button>
+      </a>
+    </div>
+  </div>
+</body>
+
+<script>
+  function retreat() {
+    window.history.back(); // Either backup,
+    window.location.href = "https://weather.codingcorner.club/"; // or get sent to the web app
+  }
+</script>
+
+<style>
+  body {
+    background-color: black;
+  }
+
+
+
+  .welcome-text{
+  color: #00ff00;
+  position: absolute;
+  left: 50%;
+  top: 25%;
+  transform: translate(-50%, -50%);
+  padding: 20px;
+  line-height: 30px;
+
+  font-size: 4em;
+  font-family: 'Courier New', Courier, monospace;
+  white-space: nowrap;
+  overflow: hidden;
+  width: 16em;
+  animation: type 4s steps(60, end); 
```

Comment:
Whitespace and indentation :snake: –  [Prettier](https://prettier.io/) plugin for your editor should auto-correct this.